grafana/k6-docs
# fix: Sidebar dropdown fix

**Describe** what changes this pull request brings:
This pull request adds Sidebar dropdown fix for Docs.
sidebar navigation dropdown is getting closed on click outside of it.

**Steps to test**
1. Open the [page]()
2. Confirm that everything looks and works as expected.
